File: Add2No.java
import java.util.*;

class Mathematic
{
    public int Addition(int No1,int No2)
    {
        int iResult = 0;

        iResult = No1+No2;

        return iResult;
    }
}

class Add2No
{
    public static void main(String arg[])
    {
        Scanner sobj=new Scanner(System.in);

        int iNo1=0;
        int iNo2=0;
        int iRet=0;

        System.out.println("Enter the first no:");
        iNo1 = sobj.nextInt();

        System.out.println("Enter the second no:");
        iNo2 = sobj.nextInt();

        Mathematic mobj = new Mathematic();
        iRet = mobj.Addition(iNo1,iNo2);

        System.out.println("Addition of two numbers is: "+iRet);


    }

}

File: Add2No.c
#include<stdio.h>

int Add2No(int No1,int No2)
{
    int iResult = 0;

    iResult = No1+ No2;

    return iResult;
}
int main()
{
    int iNo1 = 0;
    int iNo2 = 0;
    int iRet = 0;

    printf("Enter the first number:\n");
    scanf("%d",&iNo1);

    printf("Enter the second number:\n");
    scanf("%d",&iNo2);

    iRet= Add2No(iNo1,iNo2);

    printf("Addition of two numbers is:%d\n",iRet);

    return 0;
}